packages/amazonq/src/extension.ts

```diff
@@ -118,11 +117,8 @@ export async function activateAmazonQCommon(context: vscode.ExtensionContext, is
     }
     // This contains every lsp agnostic things (auth, security scan, code scan)
     await activateCodeWhisperer(extContext as ExtContext)
-    if (Experiments.instance.get('amazonqLSP', false)) {
-        await activateAmazonqLsp(context)
-    } else {
-        await activateInlineCompletion()
-    }
+    Experiments.instance.get('amazonqLSP', false)
```

Comment:
why is this being called? dose `Experiments.instance.get()` have side effects?

Reply:
this is a incorrectly resolved merge conflict